### PR TITLE
new_audit(accessibility): add accessibility manual audits

### PR DESCRIPTION
**Pull request by robdodson**
Adds a set of manual checks for a11y and language to indicate that lighthouse's automated tests to not ensure total accessibility.

image

--------

_edited by paulirish_: add new screenshot using proper manual grouping

### CODE REVIEW

File: lighthouse-core/config/default.js [hunk 1]
```diff
@@ -199,6 +209,10 @@ module.exports = {
       title: 'Meta Tags Used Properly',
       description: 'These are opportunities to improve the user experience of your site.',
     },
+    'manual-a11y-checks': {
+      title: 'Manual checks to verify',
+      description: 'These checks address areas which an automated testing tool cannot cover. They do not affect your score but it\'s important that you verify them manually. Learn more in our guide on [conducting an accessibility review](https://developers.google.com/web/fundamentals/accessibility/how-to-review).'
```
Comment (paulirish) [inline lighthouse-core/config/default.js]: I think this is a little wordy, though i don't have any particular suggestions. Between this and the overall a11y description you want to take another pass?

File: lighthouse-core/audits/accessibility/manual/focusable-controls.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+ */
+
+class FocusableControls extends ManualAudit {
+  /**
+   * @return {!AuditMeta}
+   */
+  static get meta() {
+    return Object.assign({
+      name: 'focusable-controls',
+      helpText: 'All interactive controls should aim to be keyboard focusable and display a focus indicator. [Learn more](https://developers.google.com/web/fundamentals/accessibility/how-to-review#start_with_the_keyboard).',
```
Comment (paulirish) [inline lighthouse-core/audits/accessibility/manual/focusable-controls.js]: Some inconsistency between:

* custom controls
* interactive controls
* custom interactive controls

can we use the same term?

File: lighthouse-core/config/default.js [hunk 3]
```diff
@@ -199,6 +209,10 @@ module.exports = {
       title: 'Meta Tags Used Properly',
       description: 'These are opportunities to improve the user experience of your site.',
     },
+    'manual-a11y-checks': {
+      title: 'Manual checks to verify',
```
Comment (paulirish) [inline lighthouse-core/config/default.js]: i know you were just copying what we already had but....  wdyt about a diff title for these?

like

> Additional checks to manually verify

> Additional items to manually check

File: lighthouse-core/config/default.js [hunk 4]
```diff
@@ -292,6 +306,16 @@ module.exports = {
         {id: 'valid-lang', weight: 1, group: 'a11y-language'},
         {id: 'video-caption', weight: 4, group: 'a11y-describe-contents'},
         {id: 'video-description', weight: 3, group: 'a11y-describe-contents'},
+        {id: 'custom-controls-labels', weight: 0, group: 'manual-a11y-checks'},
+        {id: 'custom-controls-roles', weight: 0, group: 'manual-a11y-checks'},
+        {id: 'focus-traps', weight: 0, group: 'manual-a11y-checks'},
+        {id: 'focusable-controls', weight: 0, group: 'manual-a11y-checks'},
+        {id: 'heading-levels', weight: 0, group: 'manual-a11y-checks'},
```
Comment (paulirish) [inline lighthouse-core/config/default.js]: is the order of these deliberate?

seems like its mostly: ARIA, Focus, Order and Exclusion

i dont really want to add groupings inside of Manual, but i figure they should be mostly grouped by topic. :)

Comment (robdodson) [reply lighthouse-core/config/default.js]: good call it should be. They're split amongst keyboard, screen reader, and landmarks but I think i may have just alphabetized them cuz my brain was shutting down. I can resort

File: lighthouse-core/config/default.js (hunk 1 above)
Comment (robdodson) [reply lighthouse-core/config/default.js]: I chopped out the middle sentence if that helps?

File: lighthouse-core/config/default.js (hunk 3 above)
Comment (robdodson) [reply lighthouse-core/config/default.js]: sgtm

File: lighthouse-core/audits/accessibility/manual/focusable-controls.js (hunk 2 above)
Comment (robdodson) [reply lighthouse-core/audits/accessibility/manual/focusable-controls.js]: Settled on custom interactive controls.

Comment (robdodson) [reply lighthouse-core/audits/accessibility/manual/focusable-controls.js]: Sorry meant to add:

**Comment by robdodson** [comment]
Reworded stuff to try to make it more concise. @paulirish PTAL.

**Comment by robdodson** [comment]
Thanks Paul!

giphy 1

**Pull request merged**